src/main/java/org/folio/service/pieces/PieceStorageService.java

```diff
@@ -135,6 +135,7 @@ public Future<PieceCollection> getAllPieces(String query, RequestContext request
   }
 
   public Future<PieceCollection> getAllPieces(int limit, int offset, String query, RequestContext requestContext) {
+    log.info("getAllPieces:: limit: {}, offset: {}, query: {}", limit, offset, query);
```

Comment:
thinking debug for such common get requests is more prefferable

Reply:
> thinking debug for such common get requests is more prefferable

Yeah, I think it was debug then after my investigation I forgot to revert it back, will do